docs/detailed_test_config.rst

```diff
@@ -238,6 +238,11 @@ called after it. This means that before any cleanup happens, you get a
 chance to inspect the system -- both through Teuthology and via extra
 SSH connections -- and the cleanup completes only when you choose so.
 Just exit the interactive Python session to continue the cleanup.
+The interactive-on-error could be enabled with ``--interactive-on-error`` option
+with ``teuthology`` command.
+
+Note: It is advicable to use interactive-on-error mode with ``teuthology``
```

Comment:
"advisable", but this whole Note is questionable IMO; should be obvious

Reply:
hmm, seems fair, since it's developer centric.